src/device-registry/models/Event.js

```
@@ -34,6 +35,256 @@ const AQI_RANGES = {
+// MongoDB switch case expression for AQI color
+const getAqiColorExpression = () => ({
+  $switch: {
+    branches: [
+      {
+        case: {
+          $and: [
+            { $gte: ["$pm2_5.value", "$aqi_ranges.good.min"] },
+            { $lte: ["$pm2_5.value", "$aqi_ranges.good.max"] },
+          ],
+        },
+        then: AQI_COLORS.good,
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)